docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -288,189 +293,121 @@ Settings (Windows).
+be accessible from the host machine itself.
+****
+
+[source,yaml,subs="attributes"]
 ----
-include::docker/elastic-docker-tls.yml[]
+include::docker/docker-compose.yml[]
 ----
 
-<1> Generate and apply a trial license that supports Transport Layer Security.
-<2> Enable Transport Layer Security to encrypt client communications.
-<3> Enable Transport Layer Security to encrypt internode communications.
-<4> Allow the use of self-signed certificates by not requiring hostname verification.
 endif::[]
 --
 
 [discrete]
 ==== Start your cluster with security enabled and configured
 
-. Generate certificates for {es} by bringing up the `create-certs` container:
+. Modify the `.env` file and enter strong password values for both the
+`ELASTIC_PASSWORD` and `KIBANA_PASSWORD` variables.
 +
---
-["source","sh"]
-----
-docker-compose -f create-certs.yml run --rm create_certs
-----
-
---
-
-. Bring up the three-node {es} cluster:
-+
---
-["source","sh"]
-----
-docker-compose -f elastic-docker-tls.yml up -d
-----
+NOTE: You must use the `ELASTIC_PASSWORD` value for further interactions with
+the cluster. The `KIBANA_PASSWORD` value is only used internally when
+configuring {kib}.
 
-IMPORTANT: At this point, {kib} cannot connect to the {es} cluster.
-You must generate a password for the built-in `kibana_system` user, update the `ELASTICSEARCH_PASSWORD`
-in the compose file, and restart to enable {kib} to communicate with the secured cluster.
-
---
-
-. Run the `elasticsearch-setup-passwords` tool to generate passwords for all built-in users,
-including the `kibana_system` user. If you don't use PowerShell on Windows, remove the trailing `\`characters
-and join the lines before running this command.
+. Create and start the three-node {es} cluster and {kib} instance:
 +
---
 ["source","sh"]
 ----
-docker exec es-node01 /bin/bash -c "bin/elasticsearch-setup-passwords \
-auto --batch --url https://es-node01:9200"
+docker-compose up -d
 ----
 
-IMPORTANT: Make a note of the generated passwords.
-You must configure the `kibana_system` user password in the compose file to enable {kib} to connect to {es},
-and you'll need the password for the `elastic` superuser to
-log in to {kib} and submit requests to {es}.
---
+. When the deployment has started, open a browser and navigate to http://localhost:5601[http://localhost:5601] to
+access {kib}, where you can load sample data and interact with your cluster.
 
-. Set `ELASTICSEARCH_PASSWORD` in the `elastic-docker-tls.yml` compose file to the password
-generated for the `kibana_system` user.
-+
---
-ifeval::["{release-state}"=="unreleased"]
-NOTE: Version {version} of {es} has not been released,
-so the sample compose file is not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
-endif::[]
+[[docker-compose-remove-containers]]
+[discrete]
+==== Remove Docker Compose containers
```

Comment:
```suggestion
==== Shut down and clean up the deployment
```

Reply:
I'll change this to **Stop and remove the deployment**.